sdk/communication/azure-communication-email/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0 (2023-03-31)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+The public release of the Azure Communication Services SDK for Email has the following features:
```

Comment:
This should follow the format that is shown on the left hand side with the sub-sections.

Reply:
There aren't any changes between this version and the previous preview version. Should I just leave these sections in here, but blank? I vaguely remember getting an error in the release pipeline when I left a section empty last time.